### PR TITLE
Review the new code

### PR DESCRIPTION
**Pull request by navya091999**
Review the new code

### CODE REVIEW

File: terraform.txt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  tags = {
+    Name = "tf-example"
+  }
+}
+
+resource "aws_subnet" "my_subnet" {
+  vpc_id            = aws_vpc.my_vpc.id
+  cidr_block        = "172.16.10.0/24"
+  availability_zone = "us-west-2a"
+
```
Comment (KrishnaAzure2) [inline terraform.txt:13]: update code 
now remove the space

Comment (KrishnaAzure2) [reply terraform.txt:13]: pls check the code

**Review by KrishnaAzure2 — CHANGES_REQUESTED** [review]

File: terraform.txt (hunk 1 above)
Comment (navya091999) [inline terraform.txt:13]: can u check the updated code

**Comment by KrishnaAzure2** [comment]
code looks good

**Pull request merged**